src/components/molecules/PortalList/PortalList.scss

```diff
@@ -0,0 +1,4 @@
+.PortalList {
+  margin: 0;
+  padding: 0;
```

Comment:
Is this overriding some default values already set by our global.scss or the library itself?

Reply:
Should usually be overriding browser defaults.